website/docs/language/import/index.mdx

```diff
@@ -131,3 +132,32 @@ import {
   id = "i-abcd1234"
 }
 ```
+
+### Importing multiple resources
+
+You can import multiple resources with one import block by using a `for_each` expression. This expression accepts a [set or a map](/docs/language/expressions/types/) and provides the `each.key` and `each.value` variables to access the individual elements.
+
+In the example below, you can specify a list of server IDs to be imported. If you specify an empty list, the `random_id` resource will generate all IDs randomly. If you specify some IDs, the import block will import the specified IDs and the resource will randomly generate the rest. Note, the `random_id` resource requires the IDs to be in base64 format.
+
+```
+variable "server_ids" {
+  type = list(string)
+}
+
+resource "random_id" "test_id" {
+  byte_length = 8
+  count = 2
+}
+
+import {
+  to = random_id.test_id[tonumber(each.key)]
+  id = each.value
+  for_each = {
+    for idx, item in var.server_ids: idx => item
```

Comment:
This is an interesting example (I didn't even know it works), but shouldn't we use a more common way of calling for a for_each in our docs? Something more similar to [this](https://opentofu.org/docs/language/meta-arguments/for_each/#using-sets) for example.
I don't want people to be confused.

Reply:
Makes sense.
I used the `random_integer` resource while developing the resource. But I don't think it can be a very good example:
```
locals {
    numbers = {
      "first" = "0,1,0"
      "second" = "0,1,0"
    }
}

resource "random_integer" "a" {
  for_each = local.numbers
  min =	1
  max = 50000
}
```
Overall, I don't have a strong opinion about it, so I'm leaving that decision to you.